docs/pages/dashboard/adding-sites.es.md

```diff
@@ -97,10 +97,10 @@ const AddSiteModal = ({ children }) => {
         </ModalContent>
       </Modal>
     </>
-  )
-}
+  );
```

Comment:
Could we prevent these Prettier changes?

Reply:
Yes we can .. Just need to set `semicolons: false` in the `.prettierrc.js`. 
The choice is yours as the author of the project. 
Can do another pull -> push if we have to avoid the semicolons

Reply:
We could also do something like this for the embedded code in the `*.md` files:
```md
<!-- prettier-ignore -->
Do   not    format   this
```

Reply:
Following up here - could that be addressed? 🙏